browser-test/bin/fmt

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# find absolute path to local browser-test directory
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && cd .. && pwd )"
```

Comment:
Suggestion (blocking): this looks like it's intending to make the script runnable from any working directory, but also looks like it will change the current working directory to the project's base directory. Recommend either requiring the user to be in the project's working directory when they run the script or restoring their original directory after the script completes (`pushd` and `popd` may be helpful commands for the latter approach).

Reply:
The change directory `cd` command is always run in a subshell, which doesn't change the directory of the current shell in script. Also I just tested, even if I explicitly change directly in the script, the script itself is run in a subshell so won't affect user's directory (if I understand correctly).